tests/system/action/projector/test_create.py

```diff
@@ -0,0 +1,65 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorCreateActionTest(BaseActionTestCase):
+    def test_create_correct(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV"},
+            }
+        )
+        response = self.request(
+            "projector.create",
+            {
+                "name": "test projector",
+                "meeting_id": 222,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "projector/1", {"name": "test projector", "meeting_id": 222}
+        )
+
+    def test_create_all_fields(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV"},
+            }
+        )
+        data = {
+            "name": "Test",
+            "meeting_id": 222,
+            "width": 100,
```

Comment:
As I see this test: `width`, `aspect_ratio_numerator` and `aspect_ratio_denominator` must be `>=1`. Is this possible to do in the models.yml? Please add this somewhere (the update also needs this info).

Reply:
done, but the https://github.com/OpenSlides/OpenSlides/pull/5914 with the changed models.yml is necessary, too